aws/lambdas/hhvm-get-build-status/index.js

```diff
@@ -39,21 +39,27 @@ exports.handler = async (event) => {
     'macos-mojave':
       'homebrew-bottles/'+bin_prefix+'.mojave.bottle.tar.gz',
   };
+  const unpublished_paths = {};
```

Comment:
nit: maybe rename to `scratch_paths` or something like that?

It took me a while to realize this is not actually a list of unpublished paths.

Reply:
built_but_not_published maybe?

Reply:
It's a list of paths where we will be looking for the packages in the scratch S3, right?

We don't know the status of any of the packages at this point so calling it anything like "(un)built" or "(un)published" seems misleading (it contains paths to all packages, not just the ones in the specified state).